### PR TITLE
Update Bonded Deposits terminology

### LINKED ISSUE #161: Users SBAT refer to simplified terminology in BondedDeposits
### Expected Behavior

Please describe the behavior you are expecting

### Current Behavior

What is the current behavior?

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

Rename references as follows:
* BondedDeposits -> LockedGold
* Bonded -> Locked
* Deposit(s) -> Commitment(s)
* Rebond -> Extend..
* and a few more.

### Tested

On a testnet and running through migrations and with CLI

### Other changes

* Fix a bad link in .env

### Related issues

- Fixes #161

### Backwards compatibility

NOT Backwards compatible -- contract names change.

Walletkit will not build when a contract name changes, so the build is likely to fail!

### CODE REVIEW

File: packages/celotool/src/lib/utils.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export function execCmd(
   cmd: string,
   execOptions: any = {},
   rejectWithOutput = false,
-  pipeOutput = false
+  pipeOutput = true
```
Comment (m-chrzan) [inline packages/celotool/src/lib/utils.ts]: Why this change?

File: packages/cli/src/commands/lockedgold/register.ts [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import { BondedDeposits } from '@celo/walletkit'
-
+import { LockedGold } from '@celo/walletkit'
 import { BaseCommand } from '../../base'
 import { displaySendTx } from '../../utils/cli'
 import { Flags } from '../../utils/command'
 
 export default class Register extends BaseCommand {
-  static description = 'Register an account for bonded deposit eligibility'
+  static description = 'Register an account for Locked Gold'
```
Comment (m-chrzan) [inline packages/cli/src/commands/lockedgold/register.ts:7]: Wash changing the wording a conscious choice?

File: packages/cli/src/commands/lockedgold/rewards.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
 import { flags } from '@oclif/command'
-import { BondedDepositAdapter } from '../../adapters/bonded-deposit'
+import { LockedGoldAdapter } from '../../adapters/locked-gold'
 import { BaseCommand } from '../../base'
 import { displaySendTx } from '../../utils/cli'
 import { Flags } from '../../utils/command'
 
 export default class Rewards extends BaseCommand {
-  static description = 'Manage rewards for bonded deposit account'
+  static description = 'Manage rewards for locked Gold account'
```
Comment (m-chrzan) [inline packages/cli/src/commands/lockedgold/rewards.ts]: Flagging discrepancy between here and the previous file, "Locked Gold" vs. "locked Gold". Both appear several times throughout.

File: packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md [hunk 4]
```diff
@@ -22,14 +22,14 @@ Validators are eligible for epoch payments if they proposed at least one block d
 
 - The annual cost of running and securing a validator node, including overhead: 60,000 Celo Dollar
 - Profit margin: $$20\%$$
-- Flat fee to validators : $$annual\_val\_fee = \frac{60,000}{1 - 0.2} = 75,\!000$$ Celo Dollar
+- Flat fee to validators : $$annual_val_fee = \frac{60,000}{1 - 0.2} = 75,\!000$$ Celo Dollar
```
Comment (m-chrzan) [inline packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: Are these underscores going to appear correctly? We escaped them since by default they make subscripts in LaTeX.

Comment (m-chrzan) [reply packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: Ditto multiple spots below.

File: packages/docs/celo-codebase/protocol/proof-of-stake/locked-gold.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Bonding Funds
```
Comment (m-chrzan) [inline packages/docs/celo-codebase/protocol/proof-of-stake/locked-gold.md]: "Locking", "Locking Up"?

File: packages/cli/src/commands/lockedgold/register.ts (hunk 2 above)
Comment (timmoreton) [reply packages/cli/src/commands/lockedgold/register.ts:7]: yes

File: packages/cli/src/commands/lockedgold/rewards.ts (hunk 3 above)
Comment (timmoreton) [reply packages/cli/src/commands/lockedgold/rewards.ts]: thanks - standardized on Locked Gold

File: packages/celotool/src/lib/utils.ts (hunk 1 above)
Comment (timmoreton) [reply packages/celotool/src/lib/utils.ts]: oops, reverted

File: packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md (hunk 4 above)
Comment (timmoreton) [reply packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: must be my editor :-( reverted

File: packages/docs/celo-codebase/protocol/proof-of-stake/locked-gold.md (hunk 5 above)
Comment (timmoreton) [reply packages/docs/celo-codebase/protocol/proof-of-stake/locked-gold.md]: fixed!

File: packages/cli/src/commands/lockedgold/rewards.ts (hunk 3 above)
Comment (m-chrzan) [reply packages/cli/src/commands/lockedgold/rewards.ts]: Docs still use "locked Gold".

File: packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md (hunk 4 above)
Comment (m-chrzan) [reply packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: Looks like your editor is very stubborn - doesn't appear to be reverted?

**Comment by mcortesi** [comment]
@timmoreton check the changes on registry-utils from a recent PR.

Also, take notice this PR will make cli/contractkit/walletkit unusable in our current environments until we redeploy them

**Comment by timmoreton** [comment]
@mcortesi just merged your changes in. The WalletKit dependency on a real network is a real challenge. I'm presuming once we are good to go I will upgrade integration...

File: packages/cli/import_contract_types_from_sdk.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```
Comment (ashishb) [inline packages/cli/import_contract_types_from_sdk.sh]: Where is this file being used?

File: packages/cli/src/adapters/locked-gold.ts [hunk 7]
```diff
@@ -61,13 +61,13 @@ export class BondedDepositAdapter {
     }
   }
 
-  async getBondedDepositValue(account: string, noticePeriod: string) {
+  async getLockedCommitmentValue(account: string, noticePeriod: string) {
```
Comment (ashishb) [inline packages/cli/src/adapters/locked-gold.ts:64]: How about `getLockedCommitmentValue` -> `getLockedGoldCommitmentValue` for clarity?

File: packages/cli/import_contract_types_from_sdk.sh (hunk 6 above)
Comment (timmoreton) [reply packages/cli/import_contract_types_from_sdk.sh]: good catch. it was removed by another PR and got re-added when merging

File: packages/cli/src/adapters/locked-gold.ts (hunk 7 above)
Comment (timmoreton) [reply packages/cli/src/adapters/locked-gold.ts:64]: I like it

Comment (timmoreton) [reply packages/cli/src/adapters/locked-gold.ts:64]: actually, I think the context is already clear. this is in a `LockedGold` package (as it is in `LockedGold.sol` too) and here the use of `Locked` is like `Bonded` -- contrast `getNotifiedCommitmentValue`

File: packages/docs/celo-codebase/protocol/proof-of-stake/becoming-a-validator.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Becoming a Validator
 
-To participate in the network, an operator must put up a slashable bonded deposit, register as a validator, and join a validator group. A minimum stake of one Celo Gold and a notice period of 60 days is required to be a validator in the Alfajores Testnet.
+To participate in the network, an operator must put up a slashable commitment of locked Celo Gold, register as a validator, and join a validator group. A minimum stake of one Celo Gold and a notice period of 60 days is required to be a validator in the Alfajores Testnet.
```
Comment (m-chrzan) [inline packages/docs/celo-codebase/protocol/proof-of-stake/becoming-a-validator.md:3]: "locked" here, "Locked" in governance.md above. I guess it is kind of awkward when used in a sentence, especially with the full "Celo Gold" like here.

File: packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md (hunk 4 above)
Comment (m-chrzan) [inline packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: These still need reverting.

File: packages/docs/command-line-interface/lockedgold.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+Create a locked Gold commitment given notice period and gold amount
+
+```
+USAGE
+  $ celocli lockedgold:lockup
+
+OPTIONS
+  --from=from                  (required)
+  --goldAmount=goldAmount      (required) unit amount of gold token (cGLD)
```
Comment (m-chrzan) [inline packages/docs/command-line-interface/lockedgold.md:31]: Another inconsistency - "gold" is lowercase. I guess this might be repeated elsewhere in the docs (and CLI source code). Maybe out of scope of this PR to standardize everywhere?

File: packages/docs/celo-codebase/protocol/proof-of-stake/becoming-a-validator.md (hunk 8 above)
Comment (timmoreton) [reply packages/docs/celo-codebase/protocol/proof-of-stake/becoming-a-validator.md:3]: yep! I plan to rewrite + expand this section of the docs in the near future

File: packages/docs/command-line-interface/lockedgold.md (hunk 9 above)
Comment (timmoreton) [reply packages/docs/command-line-interface/lockedgold.md:31]: out of scope I think

File: packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md (hunk 4 above)
Comment (timmoreton) [reply packages/docs/celo-codebase/protocol/proof-of-stake/epoch-rewards.md]: added to `.prettierignore` :-/

**Review by m-chrzan — APPROVED** [review]
